sdk/test/Services/S3/IntegrationTests/TransferUtilityTests.cs

```diff
@@ -2125,11 +2125,7 @@ public void AssertEventFired()
                 if (EventException != null)
                     throw EventException;
 
-                // Since  AWSSDKUtils.InvokeInBackground fires the event in the background it is possible that we check too early that the event has fired. In this case, we sleep and check again.
-                for (int retries = 1; retries < 5 && !EventFired; retries++)
-                {
-                    Thread.Sleep(1000 * retries);
-                }
+                // Since events are now fired synchronously, we can check immediately without retries
```

Comment:
this function was only used for validating the _**non**_ in progress events. theres a separate function that validates the in progress once which still has the retry logic since its a background thread